skyvern/webeye/utils/dom.py

```diff
@@ -411,6 +430,71 @@ async def select_by_index(
         await options.nth(index).click(timeout=timeout)
 
 
+class ReactSelectDropdown(AbstractSelectDropdown):
+    def __init__(self, skyvern_frame: SkyvernFrame, skyvern_element: SkyvernElement) -> None:
```

Comment:
this can be moved to the base class

Reply:
Maybe we can do it if we failed to find a general solution for the select pattern.